staff/judith-gongora/cv/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>Cv</title>
+</head>
+<body>
+        <main>
+            <article>
+                <section>
+                    <h1>Judith Góngora</h1>
+                    <dl>
```

Comment:
`dl` suits best for the definitions of things, not for listing personal data. for that you may use other tags (list, address, etc). please, check the link https://www.w3schools.com/tags/tag_dl.asp